pkg/scraper/aws/eks.go

```diff
@@ -134,7 +135,10 @@ func processCluster(ctx context.Context, awsClient interfaces.AWSClient, cluster
 
 func createK8sConfig(ctx context.Context, awsClient interfaces.AWSClient, clusterInfo *eks.DescribeClusterOutput, clusterName string) (*rest.Config, error) {
 	var rawConfig *rest.Config
-	cert, _ := base64.RawStdEncoding.DecodeString(*clusterInfo.Cluster.CertificateAuthority.Data)
+	cert, err := base64.RawStdEncoding.DecodeString(strings.TrimRight(*clusterInfo.Cluster.CertificateAuthority.Data, "="))
```

Comment:
this doesn't seem right?

Reply:
when I base64 decode, I usually use `base64.stdencoding.DecodeString`

Reply:
> RawStdEncoding is the standard raw, unpadded base64 encoding, as defined in [RFC 4648 section 3.2](https://rfc-editor.org/rfc/rfc4648.html#section-3.2). This is the same as StdEncoding but omits padding characters.

Reply:
> StdEncoding is the standard base64 encoding, as defined in [RFC 4648](https://rfc-editor.org/rfc/rfc4648.html).

Reply:
Thank you!